Jenkinsfile

```diff
@@ -624,10 +624,10 @@ podTemplate(name: 'fedora-atomic-inline', label: 'fedora-atomic-inline', cloud:
                         echo "Duffy Deallocate ran for stage ${current_stage} with option ${env.DUFFY_OP}\r\n" +
                              "DUFFY_HOST=${env.DUFFY_HOST}"
 
-//                     step([$class: 'XUnitBuilder',
-//                          thresholds: [[$class: 'FailedThreshold', unstableThreshold: '1']],
-//                          tools: [[$class: 'JUnitType', pattern: "${env.ORIGIN_WORKSPACE}/logs/ansible_xunit.xml"]]]
-//                    )
+                     step([$class: 'XUnitBuilder',
+                          thresholds: [[$class: 'FailedThreshold', unstableThreshold: '1']],
+                          tools: [[$class: 'JUnitType', pattern: "**/**/*.xml"]]]
```

Comment:
Is this overkill? I think **/*.xml will find all xml files in sub-dirs

Reply:
It might be.. I'm not exactly sure where it executes from.  If it goes from workspace, then **/*.xml should still find $WORKSPACE/logs/*.xml.  If it goes from anywhere higher, I don't think it is overkill.

Reply:
I had an issue with scrutinizing it down to the directory.  If you want @johnbieren make it to the directory we had before and we can see if it runs